### ISSUE TITLE
WARNING message about PATH being altered to include directory that is already in path

### ISSUE THREAD

**Issue by natedogith1** [ansible/ansible-lint#2048]
##### Summary

ansible-lint is giving `WARNING: PATH altered to include /usr/bin` despite that already being on the PATH.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible [core 2.12.3]
  config file = None
  configured module search path = ['/home/nfaulkne/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/nfaulkne/.local/lib/python3.8/site-packages/ansible
  ansible collection location = /home/nfaulkne/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/nfaulkne/.local/bin/ansible
  python version = 3.8.10 (default, Nov 26 2021, 20:14:08) [GCC 9.3.0]
  jinja version = 2.10.1
  libyaml = True
ansible-lint --version
WARNING: PATH altered to include /usr/bin
ansible-lint 6.0.2 using ansible 2.12.3
```

- ansible installation method: `python3 -m pip install ansible`
- ansible-lint installation method: `python3 -m pip install ansible-lint`

##### OS / ENVIRONMENT

WSL Ubuntu

##### STEPS TO REPRODUCE

```console (paste below)
ansible-lint --version
WARNING: PATH altered to include /usr/bin
ansible-lint 6.0.2 using ansible 2.12.3
```

##### Desired Behavior

The warning shouldn't occur under typical usage

##### Actual Behavior

```
echo $PATH; ansible-lint --version
/home/nfaulkne/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/lib/wsl/lib:/snap/bin
WARNING: PATH altered to include /usr/bin
ansible-lint 6.0.2 using ansible 2.12.3
```
With a shortened PATH:
```
echo $PATH; ansible-lint --version -vvv
/usr/bin:/home/nfaulkne/.local/bin
WARNING: PATH altered to include /usr/bin
ansible-lint 6.0.2 using ansible 2.12.3
```

**Comment by ziegenberg** [comment]
Also happens on the current Fedora release 35.

**Comment by gtirloni** [comment]
Same behavior in Fedora 36 with ansible-lint 5.4.0 and ansible 2.9.27

```shell
$ ansible-lint --version
WARNING: PATH altered to include /usr/bin
ansible-lint 5.4.0 using ansible 2.9.27
```

**Comment by shinebayar-g** [comment]
Getting same error 
```
❯ ansible-lint --version
WARNING: PATH altered to include /home/me/.pyenv/versions/3.10.8/envs/test/bin :: This is usually a sign of broken local setup, which can cause unexpected behaviors.
ansible-lint 6.9.0 using ansible 2.14.0
```

**Comment by ssbarnea** [comment]
@shinebayar-g You failed to run `echo $PATH` before running the linter. Version of Ansible is irrelevant here. Basically ansible-lint will warn you if you call it from a directory which is not in PATH, as that could make it accidentally call another ansible installation.

I use pyenv myself but it seems to not cause any problems on macos, but I usually do not create venvs with it.

**Comment by nervo** [comment]
@ssbarnea what about symlinks ?

Using pipx, ansible-lint is a symlink in `/usr/local/bin`, which is in - of course - $PATH :)

```shell
→ ls -lsa /usr/local/bin
...
    0 lrwxrwxrwx 1 root root       46 Feb  6 08:21 ansible-lint -> /usr/local/venvs/ansible-lint/bin/ansible-lint
...
```

```shell
→ echo $PATH
/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
```

**Comment by ssbarnea** [comment]
@nervo That has nothing to do with symlinks. I will lock this issue to prevent further noise. Feel free to use discussions if help is needed.
